src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -469,6 +471,22 @@ export default function QuestionnaireEditor({ id }: QuestionnaireEditorProps) {
           <Button variant="outline" onClick={handleCancel}>
             {t("cancel")}
           </Button>
+          {id && (
+            <TooltipComponent content={t("copy_questionnaire_to_clipboard")}>
```

Comment:
hmm, we should probably clarify that it only copies over the saved questionnaire 🤔 

Either that or tie to a dirty form state, but then we will have to add logic for keeping track of that 😅